jerry-core/parser/js/js-parser.c

```diff
@@ -1370,6 +1376,13 @@ parse_print_final_cbc (ecma_compiled_code_t *compiled_code_p, /**< compiled code
     } \
   } while (0)
 
+/**
+ * TODO
+ *
+ * @param page_p TODO
+ * @param offset TODO
+ * @param real_offset TODO
+ */
```

Comment:
Forward iterator: move to the next byte code. Also updates the offset of the previous byte code.